dotnet/api/Tests/Business/Conductors.Tests/xunit.runner.json

```diff
@@ -1,4 +1,4 @@
 {
-    "parallelizeAssembly": false,
-    "parallelizeTestCollections": false
+    "parallelizeAssembly": true,
+    "parallelizeTestCollections": true
 }
```

Comment:
Nitpick: mind throwing a new line in this file? (I think the other json files are the same, so won't spam ya w/ comments)